Slash/Utility/Comments/Comments.pm

```diff
@@ -1198,47 +820,22 @@ sub printComments {
 		cid		=> $cid,
 		pid		=> $pid,
 		cc		=> $cc,
-		lcp		=> $lcp,
 		lvl		=> $lvl,
-		discussion2	=> $discussion2,
 		anon_dump	=> $anon_dump,
 	}, { Return => 1 });
 
-#slashProf("getCommentTextCached", "printCommComments");
 	# We have to get the comment text we need (later we'll search/replace
 	# them into the text).
-	my $comment_text = $slashdb->getCommentTextCached(
-		$comments, [ grep { !$comments->{$_}{dummy} } @{$user->{state}{cids}} ],
-		{ mode => $form->{mode}, cid => $form->{cid}, discussion2 => $discussion2 }
-	);
-#slashProf("comment regexes", "getCommentTextCached");
```

Comment:
I wonder if they did the search and replace  so they could do the other manipulations on a smaller data structure